### PR TITLE
[WebXR Hand input] Add setPath on the XRHandModelFactory

### PR DESCRIPTION
**Pull request by fernandojsg**
As other loaders works

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

**Comment by dyarosla** [comment]
Thanks for fixing the broken samples! Works great now.

File: examples/jsm/webxr/XRHandOculusMeshModel.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ class XRHandOculusMeshModel {
 		var loader = new FBXLoader();
 		const low = options && options.model === "lowpoly" ? "_low" : "";
 
-		loader.load( `../../models/fbx/OculusHand_${handedness === "right" ? "R" : "L"}${low}.fbx`, object => {
+		loader.setPath( path );
+		loader.load( `fbx/OculusHand_${handedness === "right" ? "R" : "L"}${low}.fbx`, object => {
```
Comment (mrdoob) [inline examples/jsm/webxr/XRHandOculusMeshModel.js:15]: Why `fbx/`?

Comment (dyarosla) [reply examples/jsm/webxr/XRHandOculusMeshModel.js:15]: I believe it's because they're currently stored under examples/models/fbx

Comment (mrdoob) [reply examples/jsm/webxr/XRHandOculusMeshModel.js:15]: https://github.com/mrdoob/three.js/commit/7cb0a77f29fc4fd9e11485a235b82b27d7d8a31e

**Pull request merged**